### ISSUE TITLE
Filtered results should be sorted by event created date

### ISSUE THREAD

**Issue by charlesbrandt** [City-of-Bloomington/road_closings#70]
Currently, the events for a given set of filters are sorted by the start date. We received feedback that it would be more useful to see those events that were added most recently first. It looks like the Google Calendar API provides a "created" attribute for events.

**Comment by inghamn** [comment]
I don't think this should be a constant thing.  Seeing events in the order that they're scheduled is important, too.

It sounds like we should show the created date in the display, and provide a sorting selector that lets the user sort based on whatever they want to sort by.

**Comment by rbdietz** [comment]
created date makes no sense to public users.  Do not change this until
there is some discussion about sensible ordering.

**Comment by rbdietz** [comment]
Is there a better way to sort/display current road events?

**Comment by adabutch** [comment]
Date filters don't seem to be working when accessing via JSON format:

**Blocks issue:** https://github.com/City-of-Bloomington/myBloomington/issues/45

https://bloomington.in.gov/inroads/events?format=json&start=2019-12-10&end=2019-12-11&eventTypes%5B%5D=roadClosed&eventTypes%5B%5D=localOnly&eventTypes%5B%5D=laneRestriction&eventTypes%5B%5D=reservedMeter&eventTypes%5B%5D=noisePermit&eventTypes%5B%5D=sidewalk
